autoload/neomake/makers/ft/ruby.vim

```diff
@@ -1,7 +1,7 @@
 " vim: ts=4 sw=4 et
 
 function! neomake#makers#ft#ruby#EnabledMakers()
-    return ['mri', 'rubocop', 'reek']
+    return ['mri', 'rubocop', 'reek', 'rubylint']
```

Comment:
@wsdjeg 
> and also you can provide a g:var for user to set the executable.

That would be `g:neomake_ruby_rubylint_exe` (already supported).  See https://github.com/neomake/neomake/blob/1b585580f2f1d33dc115ebf86466bbb4112ed8da/doc/neomake.txt#L186-L190.